api/user/author/model.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import event
+
+from api import db
+
+
+@dataclass
+class Author(db.Model):
+    id: int = db.Column(db.Integer, primary_key=True)
+    url: str = db.Column("url", db.Text, nullable=True)
+    host: str = db.Column("host", db.Text, nullable=False)
+    displayName: str = db.Column("displayName", db.String(20), nullable=False)
+    github: str = db.Column("github", db.Text, nullable=False)
+    profileImage: str = db.Column("profileImage", db.Text, default="")
+
+
+@event.listens_for(Author, "after_insert")
```

Comment:
Since ID is autogenerated this is a basic setup to have url be generated based on autogenerated id